openerp-atomfeed-service/src/main/java/org/bahmni/feed/openerp/worker/OpenERPSellableServiceWorker.java

```diff
@@ -0,0 +1,59 @@
+package org.bahmni.feed.openerp.worker;
+
+import org.bahmni.feed.openerp.ObjectMapperRepository;
+import org.bahmni.feed.openerp.client.OpenMRSWebClient;
+import org.bahmni.feed.openerp.domain.OpenMRSResource;
+import org.bahmni.openerp.web.client.OpenERPClient;
+import org.bahmni.openerp.web.request.OpenERPRequest;
+import org.bahmni.openerp.web.request.builder.Parameter;
+import org.ict4h.atomfeed.client.domain.Event;
+import org.ict4h.atomfeed.client.service.EventWorker;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+
+public class OpenERPSellableServiceWorker implements EventWorker {
```

Comment:
We don't know whether the resource is a Service or not. I would suggest to keep generality and rename to "OpenERPSellableResourceWorker" or "OpenERPSellableProductWorker" or ""OpenERPSellableTypeWorker"